clients/src/main/java/org/apache/kafka/clients/consumer/internals/CoordinatorRequestManager.java

```diff
@@ -53,24 +51,27 @@
 public class CoordinatorRequestManager implements RequestManager {
     private static final long COORDINATOR_DISCONNECT_LOGGING_INTERVAL_MS = 60 * 1000;
     private final Logger log;
-    private final BackgroundEventHandler backgroundEventHandler;
     private final String groupId;
 
     private final RequestState coordinatorRequestState;
     private long timeMarkedUnknownMs = -1L; // starting logging a warning only after unable to connect for a while
     private long totalDisconnectedMin = 0;
     private Node coordinator;
+    // Hold the last fatal error received. It is exposed so that managers requiring a coordinator can access it and take 
```

Comment:
Super nit picky, but can we change the term _last_ to either _latest_ or _most recent_?